Openai-codex-best-practices-2026.html

```
@@ -794,7 +805,7 @@
 
     <div class="topbar">
       <span class="brand">CODEX / BEST PRACTICES</span>
-      <button id="navToggle" type="button" aria-label="目次を開く" aria-expanded="false">☰ 目次</button>
+      <button id="navToggle" aria-label="目次を開く">☰ 目次</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)